### ISSUE TITLE
Add Alt Text to Social Icon

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#283]
**User Story:**
As Spark we want our Social Icons to have Alt Text so that they will be described to visitors who are unable to see them.

**Notes:**
November Audit on RM
All Packages

**AC:**
All Packages
- Social icons in footer will have alt text

**Comment by ssabo1** [comment]
this is an issue for RM Tech, not Spark.
